src/unpack.js

```diff
@@ -138,10 +156,11 @@ function getThriveExecutableName(){
 
         return "Thrive.exe";
 
-    } else if(os.platform() === "linux"){
+    } else if(os.platform() === "linux" || "darwin"){
```

Comment:
This line probably doesn't do the intended thing. Looks like the linter caught this.

Reply:
should I just make Mac have it's own conditional again then? linux and mac call the .exe the same thing.

Reply:
JS doesn't support checking multiple conditions like that...
You need to write out the entire thing: `os.platform() === "linux" || os.platform() === "darwin"`